Shuttle-back/src/main/java/com/shuttle/ProfileChangeRequest/ProfileChangeRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.shuttle.ProfileChangeRequest;
```

Comment:
Package name should begin with a lowercase letter. Also, I think this could be moved to the `user` package.